### ISSUE TITLE
Disconnection notice not sent

### ISSUE THREAD

**Issue by etkirsch** [NullVoxPopuli/meshchat#14]
I never know when you go offline because I don't receive a disconnection. Perhaps this is not necessarily true for meshchat (and thus should go to rumby instead) but somewhere this isn't happening.

**Comment by NullVoxPopuli** [comment]
Yeah, I get it -- it's really a courtesy thing.

Looking at the logic that sends the disconnect:
```ruby
      def send_disconnect
        Meshchat::Command::SendDisconnect.new('/senddisconnect', _message_dispatcher)
      end
```

it actually isn't getting sent. I just create the object. lol

**Comment by etkirsch** [comment]
hahaha

**Comment by etkirsch** [comment]
But yeah I was sending you messages and you weren't getting them because you'd disconnected lol

**Comment by NullVoxPopuli** [comment]
yeah. I need to do some work on the relay to notify us of disconnection as well.. cause that will handle the "computer crash scenario"

**Comment by etkirsch** [comment]
Perhaps something with the ping messages?

**Comment by NullVoxPopuli** [comment]
idk. I _think_ I can figure out who is subscribed to a particular UID. And I know the relay knows about when the connection breaks.

**Comment by NullVoxPopuli** [comment]
Fixed in #13, will close when merged and tested.

**Comment by NullVoxPopuli** [comment]
merged #13
